pkg/ckecli/cmd/reboot_queue_is_enabled.go

```diff
@@ -0,0 +1,29 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/cybozu-go/well"
+	"github.com/spf13/cobra"
+)
+
+var rebootQueueIsEnabledCmd = &cobra.Command{
+	Use:   "is-enabled",
+	Short: "show reboot queue status",
+	Long:  `Show reboot queue status.`,
```

Comment:
The word "status" seems ambiguous.

```suggestion
	Long:  `Show whether the processing of the reboot queue is enabled or not.  "true" if enabled.`,
```